pkg/patch/repo.go

```diff
@@ -98,64 +99,79 @@ func (p *Repo) ApplyManyInNewBranch(patches []config.PackageRepoPatch) error {
 		return err
 	}
 
-	p.logDiff()
 	return nil
 }
 
-func (p *Repo) PublishChangesUnlessDryRun() (github.PullRequest, error) {
+func (p *Repo) PublishChangesUnlessDryRun(commit git.Commit) (github.PullRequest, error) {
```

Comment:
I think it'd be useful to add some documentation to this function and the one below because their naming is very similar. Something to make sure you use the correct one.

Reply:
Added